src/backend/core/templates/emails/_invitation_layout.html

```
@@ -0,0 +1,66 @@
+{% comment %}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)